staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -348,10 +348,12 @@ func getCurrentCertificateOrBootstrap(
 
 	currentCert, err := store.Current()
 	if err == nil {
-		// if the current cert is expired, fall back to the bootstrap cert
-		if currentCert.Leaf != nil && time.Now().Before(currentCert.Leaf.NotAfter) {
+		if currentCert.Leaf != nil && time.Now().Before(currentCert.Leaf.NotAfter) && time.Now().After(currentCert.Leaf.NotBefore) {
 			return currentCert, false, nil
 		}
+		// if the current cert is expired, fall back to the bootstrap cert
+		klog.V(4).Infof("Force to rotate certificate because the current time is not within the certificate interval")
+		return nil, true, nil
```

Comment:
drop these lines, to fall back to the bootstrap cert, we should continue to the code below

Reply:
This is my mistake. If the bootstrap is not used, it will rotate.
Done.